airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml

```diff
@@ -16,46 +16,132 @@ acceptance_tests:
         backward_compatibility_tests_config:
           disable_for_version: "2.0.0"
   basic_read:
-    tests:
-      - config_path: "secrets/config.json"
-        empty_streams:
-          - name: "bank_accounts"
-            bypass_reason: "This stream can't be seeded in our sandbox account"
-          - name: "external_account_bank_accounts"
-            bypass_reason: "This stream can't be seeded in our sandbox account"
-          - name: "external_account_cards"
-            bypass_reason: "This stream can't be seeded in our sandbox account"
-          - name: "checkout_sessions"
-            bypass_reason: "This stream can't be seeded in our sandbox account"
-          - name: "checkout_sessions_line_items"
-            bypass_reason: "This stream can't be seeded in our sandbox account"
-          - name: "transfers"
-            bypass_reason: "This stream can't be seeded in our sandbox account"
-        expect_records:
-          path: "integration_tests/expected_records.jsonl"
-          extra_fields: no
-          exact_order: no
-          extra_records: yes
-        ignored_fields:
-          invoices:
-            - name: invoice_pdf
-              bypass_reason: "URL changes upon each request for privacy/security"
-            - name: hosted_invoice_url
-              bypass_reason: "URL changes upon each request for privacy/security"
-          charges:
-            - name: receipt_url
-              bypass_reason: "URL changes upon each request for privacy/security"
-            - name: receipt_number
-              bypass_reason: "Fast changing data"
-            - name: network_token
-              bypass_reason: "Randomly added to the record"
-          payment_intents:
-            - name: receipt_url
-              bypass_reason: "URL changes upon each request for privacy/security"
-            - name: charges/data/*/receipt_url
-              bypass_reason: "URL changes upon each request for privacy/security"
-            - name: network_token
-              bypass_reason: "Randomly added to the record"
+    - config_path: "secrets/config.json"
```

Comment:
@alafanechere I'm working on testing this community PR... and I can't get the YAML right.  Help!